src/main/resources/config.yml

```diff
@@ -18,6 +18,9 @@ options:
   log-duplicate-block-entries: true
   burn-players-when-radioactive: true
   drop-excess-sf-give-items: false
+  disable-per-world-config: false
+  disabled-worlds:
+    - 'THIS_WORLD_IS_DISABLED'
```

Comment:
I am not a big fan of the `disabled-worlds` list right here as it is kinda redundant with what World Settings try to achieve which is not really comfortable.

Reply:
I put it there in case some one decides not to use the base system, where the world file is no longer present